### PR TITLE
Fix this was me challenge

### PR DESCRIPTION
**Pull request by returnWOW**
from this bug: https://github.com/stanvanrooy/instauto/issues/76
I try to fix this, and local tested.
When server only support challenge choice 1, this situation some times need other device logged account to confirm to send verification code.
From this bug I found another issue, to save auth data in func  `save_to_disk`: json serializable error.

### CODE REVIEW

File: instauto/api/actions/challenge.py [hunk 1]
```diff
@@ -22,27 +22,43 @@ class ChallengeMixin:
 
     def _handle_challenge(self, resp: requests.Response) -> bool:
         resp_data = resp.json()
+        print("auth data1\n{}".format(resp_data))
```
[the remark this answers is not in the document]
Comment (returnWOW) [reply instauto/api/actions/challenge.py]: Hello, This one is removed.

**Review by returnWOW — COMMENTED** [review]
No error.

**Comment by stanvanrooy** [comment]
Thanks for the help!

**Comment by stanvanrooy** [comment]
@all-contributors please add @returnWOW for code

**Pull request merged**